src/components/CompactVoiceRecorder.tsx

```
@@ -0,0 +1,287 @@
+        <MicOff className="h-4 w-4" />
+      ) : (
+        <Mic className={`h-4 w-4 ${!isProUser ? 'text-slate-400' : ''}`} />
+      )}
+    </button>
+  );
+};
+
+// Type declarations for Web Speech API
+declare global {
+  interface Window {
+    SpeechRecognition: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)